models/models.js

```diff
@@ -1,16 +1,23 @@
 const db = require('../db/connection')
+const fs = require('fs/promises')
 const format = require('pg-format')
+const endpoint = require('../endpoints.json')
 
 
-
-selectTopics = () => {
+const selectTopics = () => {
     return db.query("SELECT * FROM topics")
     .then((data) => {
-        console.log(data, 'models')
         return data.rows
     })
 }
 
+// `${__dirname}/../endpoints.json`
 
+const selectEndpoint = () => {
+    return fs.readFile(`${__dirname}/../endpoints.json`, 'utf8').then((data) => {
+        const parsedData = JSON.parse(data)
+        return parsedData
```

Comment:
could just require endpoints.json in and then just send as response in the model.